Here, we show that splicing can also contribute to histone modification, which implies bidirectional communication between epigenetic mechanisms and RNA processing.
Trimethylation of histone H3 on lysine 4 localizes near the 5 ' region of genes and is tightly associated with active loci.
Genome-wide analysis of histone methylation in human cell lines and mouse primary T cells reveals that intron-containing genes are preferentially marked with histone H3 Lys36 trimethylation relative to intronless genes.
MOTIVATION: In addition to alternative splicing, alternative polyadenylation has also been identified as a critical and prevalent regulatory mechanism in human gene expression.
the position of nucleosomes in the interior of genes has been thought to be largely random.
We propose that splicing is mechanistically coupled to recruitment of HYPB/Setd2 to elongating RNA polymerase II.
A chromatin code appears to mark introns and exons with distinct patterns of nucleosome enrichment and histone methylation.
Moreover, recent observations indicate that chromatin structure, transcription and splicing are functionally intertwined, and that modified nucleosomes with trimethylation of lysine 36 in histone subunit 3 are enriched at internal exons and the downstream flanking intronic regions of highly expressed genes.
Several lines of recent evidence support a role for chromatin in splicing regulation.
.
